_posts/2021-12-29-#20827.md

```diff
@@ -10,15 +10,54 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- When pruning is enabled in Bitcoin Core, data about old blocks is deleted to limit disk space usage.
+Users can configure a pruning target defining how much disk space to use for block and undo data.
+The minimum target is 550MB.
+
+- Bitcoin Core keeps a write buffer of UTXOs (aka dbcache).
+If the buffer didn't exist, creating a UTXO and deleting a UTXO would both cause a write operation to disk.
+As UTXOs are often short lived, modifying the buffer is a lot faster than writes to disk.
+Reading from the buffer is also cheaper than looking UTXOs up on disk.
+The buffer is flushed to disk, for example, when it grows too large.
+Depending on the buffer size, flushes can take a while.
+
+- Node operators can control the buffer size with the `-dbcache=<size in MB>` argument.
+A larger buffer takes up more system memory but takes longer to fill and thus requires fewer flushes.
+This speeds up the initial block download (IBD).
+
+- Pruning is a reason for us to flush the dbcache regardless of its memory usage.
+The maximum configured dbcache size is often not reached.
+
+- This PR changes the pruning behavior.
+Previously, we'd prune just enough files for us to be able to continue the IBD.
+We now aggressively prune all prunable files enabling us to continue with IBD without having to prune again too soon.
+Fewer prunes also mean fewer dbcache flushes, potentially speeding IBD for pruned nodes up.
+Higher dbcache sizes can be reached before the dbcache is flushed.
+
+<p style="text-align:center"><a href="../assets/img/20827-pruning.drawio.png"><img src="../assets/img/20827-pruning.drawio.png" style="max-width:100%; height:auto"></a></p>
+
+
+- PR [#12404](https://github.com/bitcoin/bitcoin/pull/12404) attempted aggressive pruning too, but was closed in favor of PR [#11658](https://github.com/bitcoin/bitcoin/pull/11658).
+PR #11658 added 10% of the prune target to the `nBuffer`.
+This is being overwritten by this PR.
+
+
 ## Questions
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
```

Comment:
Doesn't need to be a question here, but I recommend asking "what does this PR do" during the meeting, especially since the diff is so small.